README.md

```diff
@@ -1,7 +1,7 @@
 # admiral 👩‍✈️🚢⚓️ #
 
 [![GitHub Build Status](https://github.com/cisagov/admiral/workflows/build/badge.svg)](https://github.com/cisagov/admiral/actions)
-[![CodeQL](https://github.com/cisagov/admiral/workflows/CodeQL/badge.svg)](https://github.com/cisagov/admiral/actions/workflows/codeql-analysis.yml)
+[![CodeQL](https://github.com/cisagov/admiral/workflows/CodeQL/badge.svg)](https://github.com/cisagov/admiral/actions/workflows/codeql-analysis.yaml)
```

Comment:
You got a little overzealous with your search and replace.  This file should remain called `codeql-analysis.yml` for now - if it ever changes, that will be done in our skeletons:
```suggestion
[![CodeQL](https://github.com/cisagov/admiral/workflows/CodeQL/badge.svg)](https://github.com/cisagov/admiral/actions/workflows/codeql-analysis.yml)
```